src/components/Footer.css

```diff
@@ -20,12 +26,14 @@ footer .icon-container {
   font-size: 1.1em;
 }
 
-footer .icon .footer-icon {
-  max-width: 18px;
-  margin-right: 5px;
+@media all and (min-width: 767px) {
```

Comment:
this would also make the description a bit wider and better readable :)

Reply:
So, this change is for actually for the icons class rather than for the footer content. I break it into 2 because I somehow is not be able to override the margin-right property on the mobile view. But you're right about we might need to work around a better readable view for the description. I'll see what I can do for that.